app/helpers/arclight_helper.rb

```diff
@@ -6,9 +6,15 @@ module ArclightHelper
   ##
   # @param [SolrDocument]
   def parents_to_links(document)
-    safe_join(document_parents(document).map do |parent|
+    breadcrumb_links = []
+
+    breadcrumb_links << build_repository_link(document)
+
+    breadcrumb_links << Arclight::Parents.from_solr_document(document).as_parents.map do |parent|
```

Comment:
```suggestion
    breadcrumb_links << document_parents(document).map do |parent|
```
Can we reuse `document_parents(document)` here?

Reply:
updated per your request to use document_parents(document)